src/main/java/se/liu/ida/hefquin/engine/wrappers/lpgwrapper/impl/NodeMappingToURIsImpl.java

```diff
@@ -0,0 +1,26 @@
+package se.liu.ida.hefquin.engine.wrappers.lpgwrapper.impl;
+
+import org.apache.jena.graph.Node;
+import org.apache.jena.graph.NodeFactory;
+import se.liu.ida.hefquin.engine.wrappers.lpgwrapper.data.impl.LPGNode;
+
+public class NodeMappingToURIsImpl implements NodeMapping{
+
+    protected final String NS = "https://example.org/";
+    protected final String NODE = "node/";
```

Comment:
Can you please make just one string out of these two and, then, add a constructor to this class via which this string can be passed to the class (i.e., similar to how the `label` IRI is passed to `LPG2RDFConfigurationImpl`).